### ISSUE TITLE
Get command not checking if user is logged in

### ISSUE THREAD

**Issue by joshirio** [meganz/MEGAcmd#19]
```
[user@linux ~]$ mega-exec get a.txt a.txt
[err: 10:57:34] Couldn't find file
[user@linux ~]$ mega-exec put a.txt a.txt
[err: 10:57:38] Not logged in.
[user@linux ~]$ mega-exec ls
[err: 11:00:06] Not logged in.
```

Most of the commands fail if there's no session active (not logged in), but when using `get` it fails to find the file. I understand, that you can get public or shared links without the need to log in, but It would help a lot if it could check whether or not the requested file is a valid link (no permission required) and if it is not it could fail with the usual message (not logged in).

When calling MEGAcmd, at this point I really don't know if the file is not found or just the session or the user logged out.

**Comment by polmr** [comment]
Your assumption is right @joshirio, we'll fix the error message to match your expectations. Thanks for pointing that out.

**Comment by polmr** [comment]
This should have been fixed in version 0.9.9
